common/ProfileZone.hpp

```diff
@@ -91,7 +95,7 @@ class ProfileZone
 
     ~ProfileZone()
     {
-        if (m_nCreateTime.time_since_epoch() > std::chrono::time_point<std::chrono::system_clock>().time_since_epoch())
```

Comment:
Wow - we had a system-call for every profile zone destructor?
I really want the ProfileZone stuff to be almost a no-op / zero performance impact when it is not enabled. This is better - but - I'm amazed to see that thing[!] even as it disappears =)

Reply:
Not sure I see any system call there. And anyway, "we had" is an odd expression to use for code that has been there for maybe a week, and has not been used at all.

Reply:
You're quite right; I had assumed one of these would have done a gettimeofday or equivalent; but I'm wrong =)